src/components/MoneyRequestHeader.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import HeaderWithCloseButton from './HeaderWithCloseButton';
+import iouReportPropTypes from '../pages/iouReportPropTypes';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+import participantPropTypes from './participantPropTypes';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import CONST from '../CONST';
+import withWindowDimensions from './withWindowDimensions';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: iouReportPropTypes.isRequired,
+
+    /** The policies which the user has access to and which the report could be tied to */
+    policies: PropTypes.shape({
+        /** Name of the policy */
+        name: PropTypes.string,
+    }).isRequired,
+
+    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+
+    isSingleTransactionView: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    isSingleTransactionView: false,
+};
+
+const MoneyRequestHeader = (props) => {
+    const formattedAmount = props.numberFormat(props.report.total / 100, {
+        style: 'currency',
+        currency: props.report.currency,
+    });
+    const isSettled = /* ReportUtils.isSettled(props.report.reportID); */ false;
+    const workspaceName = ReportUtils.getPolicyName(props.report, props.policies);
+    const workspaceAvatar = ReportUtils.getWorkspaceAvatar(props.report, props.policies);
+    return (
+        <View style={[
+            {backgroundColor: themeColors.highlightBG},
+            styles.pl0,
+        ]}
+        >
+            <HeaderWithCloseButton
+                shouldShowAvatarWithDisplay
+                shouldShowThreeDotsButton={!isSettled}
```

Comment:
We should also show the green checkmark next to the amount if `isSettled = true`